website/snippets/quickstarts/change-way-model-materialized.md

```diff
@@ -54,10 +54,11 @@ By default, everything gets created as a view. You can override that at the dire
 
     </File>
 
-4. Enter the `dbt run` command. Your model, `customers` should now build as a view.
+4. Enter the `dbt run` command. Your model, customers should now build as a view. 
```

Comment:
Not sire if we removed the backticks on purpose?
```suggestion
4. Enter the `dbt run` command. Your model, `customers`, should now build as a view. 
```

Reply:
oh i removed it first and then copied it from the page. thank you for catching!